mooncake-store/include/master_metric_manager.h

```diff
@@ -133,7 +140,10 @@ class MasterMetricManager {
     ylt::metric::counter_t mount_segment_failures_;
     ylt::metric::counter_t unmount_segment_requests_;
     ylt::metric::counter_t unmount_segment_failures_;
+    ylt::metric::counter_t remount_segment_requests_;
+    ylt::metric::counter_t remount_segment_failures_;
     ylt::metric::counter_t ping_requests_;
+    ylt::metric::counter_t ping_failures_;
```

Comment:
We need a metric that shows the number of live clients currently in the system.

Reply:
OK. I added the active client metrics. And I also added a condition check to only show the Pings and active clients metrics when the enable_ha is set to ON.